### DISCUSSION TITLE
When will stable RC be released?

### DISCUSSION THREAD

**Discussion by webcoreinc** [GrapesJS/grapesjs#5752] [Q&A]
We use Mailwizz and GrapeJS is built in.

The builder is full of bugs and, according to my entire dev team, is early beta at best.

Any timeline on releasing a stable version?

Our customers hate this builder and we have the tickets to show it.

I have several screen records .Will try to create posst and upload when i have time. We stopped counting bugs at one point there are so many.

Thanks

**Comment by mosh-tudor** [comment]
> Our customers hate this builder

Lol... good start :-)

**Reply by mosh-tudor** [reply]
We'd all like to have a more stable editor, but that's unrealistic considering that only one person is actively working here (in his free time) 🤷🏻‍♂️

**Comment by artf** [comment]
GrapesJS is not an application, it's a framework, so if you integrate it improperly, you get an editor full of bugs.

Do you even know the version they are using?
